.codeclimate.yml

```diff
@@ -1,42 +1,30 @@
 ---
-engines:
+version: "2"
+plugins:
   shellcheck:
     enabled: true
-  checkstyle:
-    enabled: true
-    channel: "beta"
   csslint:
     enabled: true
   duplication:
     enabled: true
     config:
-      enabled: true
-      threshold: 7
       languages:
-        - typescript
-        - javascript
-        - python
-        - php
+        - typescript:
+        - javascript:
+        - python:
+        - php:
   eslint:
     enabled: true
+    channel: "eslint-8"
     config:
       config: .eslintrc
```

Comment:
"Cannot read config file: /code/.eslintrc", nej det er da klart når der ikke ligger en .eslintrc i root ╰（‵□′）╯

Reply:
(der burde så måske ligge en i root)

Reply:
For now har jeg sat den til at tage ESlint reglerne fra client/.eslintrc

Reply:
Hmm, faktisk meget god pointe... Lad os tage den med til den store PR

Reply:
Det har også været oppe og vende [her](https://github.com/INTO-CPS-Association/DTaaS/issues/24#issue-1645503049). Jeg hører lige Prasad imorgen om hvorfor den er streget over 🤔